datetimeparser/evaluator.py

```diff
@@ -1,108 +1,24 @@
-from datetime import datetime, timedelta
 from dateutil.relativedelta import relativedelta
 from typing import Union
-from pytz import timezone
+from pytz import timezone, UnknownTimeZoneError
 
-from .enums import *
 from .baseclasses import *
+from .enums import *
```

Comment:
Sort imports please

Python Built in Packages  
Empty line  
Pip installed packages  
Empty line  
Project files